src/NServiceBus.Core/Sagas/SagaMetadata.cs

```diff
@@ -405,15 +405,11 @@ static void ValidateMapping<TMessage>(Expression<Func<TMessage, object>> message
 
                 var propertyInfo = memberExpr.Member as PropertyInfo;
 
-                const string message = "When mapping a message to a saga, the member type on the message and the saga property must match. {0}.{1} is of type {2} and {3}.{4} is of type {5}.";
```

Comment:
this is still used here: https://github.com/Particular/NServiceBus/pull/4577/files#diff-57e1a3a6d5d49724fe7b19df6ad11062R424